server/controllers/class/recommended.ts

```diff
@@ -0,0 +1,14 @@
+import { Request, Response, NextFunction } from 'express';
+import { recommendedQueries } from '../../queries';
+
+const recommended = async (req: Request, res: Response, next:NextFunction) => {
+  try {
+    const { classId } = req.params;
+    const data:any = await recommendedQueries(Number(classId));
+    res.json({ data });
```

Comment:
The convention we followed in the responses is: 
`{ msg: 'message', data: { The data to send } }`